### DISCUSSION TITLE
Differences vs ArgoCD

### DISCUSSION THREAD

**Discussion by elgalu** [fluxcd/flux2#2233] [Q&A] [answered]
Both ArgoCD and Flux2 seem like great solutions, which are the biggest differences? or how to chose one over the other?

**Comment by stefanprodan** [comment, marked as answer]
You should always chose Flux 😆 

Jokes aside, here is a non-biased comparison of the two https://thenewstack.io/gitops-on-kubernetes-deciding-between-argo-cd-and-flux/
